### ISSUE TITLE
People Picker inside a text input field

### ISSUE THREAD

**Issue by gsogol** [microsoft/fluentui#26979]
### Describe the issue:

looking for a way to use this component or mention someone inside a textarea/input field in a Teams Tabs Application that uses the FluentUI react components. 

May be a different complement exists 🤷🏼‍♂️

#### Actual behavior:

It's a separate component vs a way to mention/tag someone within a

**Comment by Hotell** [comment]
Hey! to clarify, are you asking if `@fluentui/react-compomnents` provides `PeoplePicker` control ?

**Comment by gsogol** [comment]
Yes but also how to implement either a people picker inside a textarea or a mention inside the textarea

**Comment by sopranopillow** [comment]
We currently don't offer a PeoplePicker for `@fluentui/react-components`, I'll let @paulgildea comment about future plans for it. 

About the mentions, we also don't provide that functionality out of the box for any of our versions (v0, v8, v9) the closes we have is a prototype in v0 that you could take a look at. Here's a [preview](https://fluentsite.z22.web.core.windows.net/0.66.2/prototype-mentions) and here's the [source code](https://github.com/microsoft/fluentui/tree/master/packages/fluentui/react-northstar-prototypes/src/prototypes/mentions)

**Comment by gsogol** [comment]
Thank you and unfortunate. The prototype seems interesting and is what we need. Seems like it's 3 years old. Is that the intention for prototypes to take so many years? Will it be merging with v9? How does the process work?

**Comment by sopranopillow** [comment]
We currently moved to a newer version of these components `@fluentui/react-components` also called v9. Support for bug fixes and patches is still supported for v0 (which is the version that has that prototype), but we are no longer adding features to it. We don't have in the plans to add that prototype to v9 and if we do, it would be future work. 

TLDR: The prototype there is just an example that you would probably need to reference to create something similar since we don't have a similar Component/prototype in the works at the moment.

**Comment by gsogol** [comment]
Ok thanks. My suggestion would be to rethink the process. Teams is about collaboration. Collaboration involves more than 1 person. Mentions is at the core of that. Food for thought
